File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/UnknownCryptoException.java
package org.hyperledger.indy.sdk;

public class UnknownCryptoException extends IndyException
{
	private final static String message = "An unknown crypto format has been used for a DID entity key.";

	UnknownCryptoException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/SignusApi/UnknownCryptoException.cs
namespace Hyperledger.Indy.SignusApi
{
    /// <summary>
    /// Exception thrown when an unknown crypto format is used for DID entity keys.
    /// </summary>
    public class UnknownCryptoException : IndyException
    {
        const string message = "An unknown crypto format has been used for a DID entity key.";

        internal UnknownCryptoException(int sdkErrorCode) : base(message, sdkErrorCode)
        {

        }
    }

}
